pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -0,0 +1,137 @@
+package namespace
+
+import (
+	"context"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	realclient "sigs.k8s.io/controller-runtime/pkg/client"
+	client "sigs.k8s.io/controller-runtime/pkg/client/fake"
+	//"sigs.k8s.io/controller-runtime/pkg/client/config"
+	//"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func reset(ctx context.Context, r *ReconcileNamespace) {
+	blacklistedProjects = make(map[string]bool)
```

Comment:
`blacklistedProjects` seems to be declared but not used in this file. Is this ok? I would expect it to throw an error during testing.

Reply:
Since the test is running in the `namespace` package, it's referencing the package-scope variable (defined in namespace_controller.go on line 36). The goal of this assignment is to "reinitialize" it so that subsequent tests don't interact with data in the map from previous tests.